From 9984141a812c26630aa1bb1424b4f3c120c7ffe0 Mon Sep 17 00:00:00 2001
From: Michael Henning <drawoc@darkrefraction.com>
Date: Sat, 29 Dec 2012 23:57:23 -0500
Subject: [PATCH] gutils: move G_END_DECLS to the end of gutils.h

Previously, some declarations near the bottom came after the
G_END_DECLS, causing linker errors for C++ users that called those
functions.
---
 glib/gutils.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/glib/gutils.h b/glib/gutils.h
index 15f1ed9..8102f53 100644
--- a/glib/gutils.h
+++ b/glib/gutils.h
@@ -307,8 +307,6 @@ g_bit_storage (gulong number)
 }
 #endif  /* G_CAN_INLINE || __G_UTILS_C__ */
 
-G_END_DECLS
-
 #ifndef G_DISABLE_DEPRECATED
 
 /*
@@ -372,4 +370,6 @@ const gchar *g_get_tmp_dir_utf8          (void);
 gchar       *g_find_program_in_path_utf8 (const gchar *program);
 #endif
 
+G_END_DECLS
+
 #endif /* __G_UTILS_H__ */
-- 
1.8.0.3

